brave/browser/brave_browser_context.cc

```diff
@@ -507,6 +500,49 @@ content::PermissionManager* BraveBrowserContext::GetPermissionManager() {
   return permission_manager_.get();
 }
 
+void BraveBrowserContext::LaunchTorProcess(base::WaitableEvent* tor_launched,
```

Comment:
it seems odd to me to have these options/methods here. Why wouldn't they be part of the tor launcher?

Reply:
this is used to blocked the execute real launching process in separate thread while the other thread is blocked waiting for launching result.